common/data_refinery_common/models/models.py

```diff
@@ -875,6 +881,11 @@ def __str__(self):
     # Compendia details
     quant_sf_only = models.BooleanField(default=False)
     is_compendia = models.BooleanField(default=False)
+    svd_algorithm = models.CharField(
+        max_length=255,
+        choices=SVD_ALGORITHM_CHOICES,
+        default="NONE"
```

Comment:
Do you want to make the default the one we have been using (ARPACK)? If not, we should be very sure there's no code elsewhere in the project that calls this without specifying the choice.